pkg/cc_dist_library.bzl

```diff
@@ -134,12 +134,15 @@ CcFileList = provider(
 )
 
 def _flatten_target_files(targets):
-    return depset(transitive = [target.files for target in targets])
+    return depset(transitive = [
+        target.files
+        for target in targets
+        # Filter out targets from external workspaces
+        if target.label.workspace_name == ""
```

Comment:
You should probably include "com_google_protobuf" here too?  Does that pick up some unwanted dependencies?

Reply:
Good point, done.